### ISSUE TITLE
Compiling for iOS issues

### ISSUE THREAD

**Issue by Ricardo1980** [zhuhaow/NEKit#166]
Hello,

I'm trying to use this lib in an iOS app.

I setup Carthage creating a file.
I run 
carthage update --no-use-binaries --platform ios
I attach the frameworks created to my app.
I run the app and it seems it is working, but when I try to use, for example, DNSServer, that's not found.
Checking the files it seems they are not completed.
Am I missing a step?
Thanks a lot.

![screen shot 2018-05-23 at 17 14 58](https://user-images.githubusercontent.com/738681/40438875-2f053164-5eb1-11e8-9948-6b16db7edcaf.png)
![screen shot 2018-05-23 at 17 15 01](https://user-images.githubusercontent.com/738681/40438881-340beefa-5eb1-11e8-8e0b-f8656486aff7.png)
![screen shot 2018-05-23 at 17 43 28](https://user-images.githubusercontent.com/738681/40438892-39422b96-5eb1-11e8-8bd8-93947ff3586a.png)

**Comment by mikeKane** [comment]
You are importing NEKit inside of the PacketTunnel?

**Comment by zhuhaow** [comment]
As @mikeKane said, are you importing NEKit?

I'm not expert on Swift or iOS development, but my guess is this header is purely used for interactions between Swift and Objective-C where only Swift methods exposed with `@objc` are there.

**Comment by Ricardo1980** [comment]
Hello,

I just created a new iPhone app in swift and at the top of the view controller I added "import NEKit"

When adding "private var dnsServer: DNSServer!" it didn't work. But for same reason, now it is working and now I can compile it. I have no idea about the reason.

Now, I'm trying to learn how to use this library, which seems what I need for my use case.

BTW, last question:

In the docs I see that one of the features is:
"A DNS server that rewrites request and response.'"

So, can I create a DNS server that blocks a list of URLs (basically, adult content, pornography....  protect children) and all the other requests forward them to the original or well known DNS server?

Thanks a lot for your help.
